### ISSUE TITLE
[FR] Camera page showing refreshed jpegs with stream only when clicked

### ISSUE THREAD

**Issue by dkmcgowan** [dwainscheeren/dwains-lovelace-dashboard#84]
Add a param for each camera on whether it should be a jpeg or a stream, you could also ask for the jpeg url or capture it from the stream.  Then when you click the camera to full screen it, it is always a stream.  If you have more than 6 cameras, the others are broken right now because a browser cannot handle more.

**Comment by dwainscheeren** [comment]
Do you got some code for me how you now have that then? Which card do you use for the cameras (not live)

**Comment by LoneWalkerWorlf** [comment]
Here is my original lovelace code for one of my cameras:
`      - aspect_ratio: 50%
        camera_image: camera.garden
        entity: camera.garden
        image: ''
        name: Garden Camera
        show_state: false
        type: picture-entity`

Here is what it looks like:
**
![image](https://user-images.githubusercontent.com/63847434/84246968-4b9a3080-aaff-11ea-960d-545dc07c162f.png)
**

Hopefully this helps!

**Comment by dwainscheeren** [comment]
This is fixed in 2.0
Example of my `cameras.yaml` in 2.0:
```yaml
cameras:
  main_menu: 'false'
  live_view: 'false'
  entities:
    - camera.oprit_camera
    - camera.garage_camera
